packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -635,4 +660,76 @@ export default class XdslAccessCtrl {
         );
       });
   }
+
+  /**
+   * Check if change mail sending status is running
+   */
+  getChangeMailSendingTask() {
+    return this.XdslAccessService.getChangeMailTasks(
+      this.$stateParams.serviceName,
+    ).then(({ data }) => {
+      const tasks = data;
+      if (tasks.length > 0) {
```

Comment:
```suggestion
      if (data.length > 0) {
```
You can use directly data no need to add a constante.